content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -1,72 +1,216 @@
+product(s) will go into.
+
+Add a line for each additional package to be used, by clicking :guilabel:`Add a line`. Entering the
+desired amount in the :guilabel:`Done` field specifies the amount of the product to go into the
+:guilabel:`Destination Package`.
+
+Use multiple packages in a single delivery
+------------------------------------------
+
+Although a delivery order is typically associated with a single package, orders can also be split
+using the :guilabel:`Detailed Operations` window. To package 10 boxes of pencils with 2 boxes of
+erasers, select the :guilabel:`Detailed Operations` icon to the right of `Box of Pencils`.
+
+The resulting window is used to detail how the 20 products `Box of Pencils`, are to be packaged.
+Fill in `10` under the :guilabel:`Done` field to package 10 products into the
+:guilabel:`Destination Package`. Start typing under this field and select the :guilabel:`Create`
+button to create a new package, `PACK0000006`. Click :guilabel:`Add a line` to assign the
+remaining products in another package, `PACK0000007`.
+
+.. image:: usage/packages-detailed-14-15.png
+   :align: center
+   :alt: Detailed operations where the amount of product going in a pack can be specified.
+
+Select :guilabel:`Detailed Operations` on the product line for `Box of Erasers` and similarly
+set 2 :guilabel:`Done` products to `PACK0000006` and `PACK0000007` each.
+
+.. image:: usage/packages-14-15-out.png
+   :align: center
+   :alt: Put in pack once the done amount matches the demand.
+
+Once all the products for each line have been entered, click :guilabel:`Confirm` to return to the
+:abbr:`DO (Delivery Order)`.
+
+The :guilabel:`Done` quantity on the delivery order updates to show the products selected in the
+various packages. When the :guilabel:`Done` amount matches the :guilabel:`Demand` amount, click the
+:guilabel:`Put In Pack` button in the bottom-left of the delivery order form. And finally, click the
+:guilabel:`Validate` button to validate the delivery order.
+
+.. tip::
+   The :guilabel:`Put In Pack` button is intended to quickly package all products in the :abbr:`DO
+   (Delivery Order)` into 1 new package. The option to rename the package, or modify its contents,
+   are available in the :guilabel:`Detailed Operations` pop-up.
+
+Clicking on the :guilabel:`Packages` smart button in the upper-right of the delivery order reveals
+a separate page, showing all the packages used in that delivery order.
+
+Packagings
+==========
+
+*Packaging* is product specific, and refers to a disposable container that holds several units of a
+specific product. Unlike packages, packagings cannot be reusable, and each specific packaging must
+be defined on the individual product form.
+
+For example, different packages for cans of soda can be configured as a 6-pack, a 12-pack, or a case
+of 36. Each flavor of soda would need a 6, 12, and 36 can packaging configured on the individual
```

Comment:
I think it would be beneficial to use the pre formatting here for the example configuration
```suggestion
of 36. Each flavor of soda would need a `6`, `12`, and `36` can packaging configured on the individual
```